.github/workflows/cd.yaml

```diff
@@ -0,0 +1,25 @@
+name: CD Workflow
+
+on: 
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-*"
+  workflow_dispatch:
+
+jobs:
+  main:
+    name: Build & Publish core
+    runs-on: "ubuntu-latest"
+    steps:      
+      - name: Checkout
+        uses: actions/checkout@v4     
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.8"
+      - name: Build & Publish to PyPi
+        uses: JRubics/poetry-publish@v2.0
```

Comment:
This is alternative to using the `poetry` commands directly, right?

Reply:
Yes that's correct.